_foundations/typography/typeface-guidance.md

```diff
@@ -3,11 +3,12 @@
   <div class="accordion-panel">
     <ul>
       <li>Use just 1 sans serif font (Open Sans). Open Sans is a sans-serif font selected for accessibility and readability.</li>
-      <li>Use monospace... [PASCAL TO FILL IN & CSS FOR FONT SPECIMEN PLEASE]</li>
+      <li>When using the SASS source refer to the <code>$base-sans-serif</code> variable to access the sans-serif font-stack.</li>
```

Comment:
Change to `Sass` rather than SASS, following what seems most commonly used: http://sass-lang.com/